code/datums/martial/security.dm

```diff
@@ -0,0 +1,118 @@
+		to_chat(owner, "<span class='warning'>You can't use [name] while you're incapacitated.</span>")
+		return
+	var/mob/living/carbon/human/H = owner
+	if (H.mind.martial_art.streak == "pressure_point_strike")
+		owner.visible_message("<span class='danger'>[owner] assumes a neutral stance.</span>", "<b><i>Your next attack is cleared.</i></b>")
+		H.mind.martial_art.streak = ""
+	else
+		owner.visible_message("<span class='danger'>[owner] assumes the Pressure Point stance!</span>", "<b><i>Your next attack will be a Pressure Point Strike.</i></b>")
+		H.mind.martial_art.streak = "pressure_point_strike"
+
+
+/datum/martial_art/security_cqc/teach(mob/living/carbon/human/H, make_temporary=0)
+	if(..())
+		to_chat(H, "<span class = 'userdanger'>You know the arts of [name]!</span>")
+		to_chat(H, "<span class = 'danger'>Place your cursor over a move at the top of the screen to see what it does.</span>")
+		forcefuldisarm.Grant(H)
+		pressurepointstrike.Grant(H)
+
+/datum/martial_art/security_cqc/on_remove(mob/living/carbon/human/H)
+	to_chat(H, "<span class = 'userdanger'>You suddenly forget the arts of [name]...</span>")
+	forcefuldisarm.Remove(H)
+	pressurepointstrike.Remove(H)
+
+/datum/martial_art/security_cqc/proc/check_streak(mob/living/carbon/human/U, mob/living/carbon/human/T)
+	switch(streak)
+		if("forceful_disarm")
+			forceful_disarm(U, T)
+			streak = ""
+			return 1
+		if("pressure_point_strike")
+			pressure_point_strike(U, T)
+			streak = ""
+			return 1
+	return 0
+
+/datum/martial_art/security_cqc/proc/forceful_disarm(mob/living/carbon/human/U, mob/living/carbon/human/T)//User is U, target is T
+	T.visible_message("<span class = 'warning'>[U] forcefully disarms [T]!</span>", "<span class = 'userdanger'>[U] forcefully disarms you!", null, COMBAT_MESSAGE_RANGE)
+	playsound(get_turf(U), 'sound/effects/grillehit.ogg', 50, 1, -1)
+	var/obj/item/activeItem = T.get_active_held_item()
+	T.dropItemToGround(activeItem)
+	var/disarmDir = get_dir(U, T)
+	var/turf/throwAt = get_ranged_target_turf(activeItem, disarmDir, 2)
+	activeItem.throw_at(throwAt, 7, 1)
+	log_combat(U, T, "Forceful Disarm")
+
+/datum/martial_art/security_cqc/proc/pressure_point_strike(mob/living/carbon/human/U, mob/living/carbon/human/T)
+	var/obj/item/bodypart/targetedZone = T.get_bodypart(ran_zone(U.zone_selected))
+	var/armorBlock = T.run_armor_check(targetedZone, "melee")
+	T.visible_message("<span class = 'warning'>[U] hits you with a pressure point strike [T]!</span>", \
+	 "<span class = 'userdanger'>[U] hits a pressure point on your [targetedZone]!", null, COMBAT_MESSAGE_RANGE)
+	playsound(get_turf(U), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	T.apply_damage(1, BRUTE, targetedZone, armorBlock)
+	T.apply_damage(20, STAMINA, targetedZone, armorBlock)
+	log_combat(U, T, "Pressure Point")
+
+/datum/martial_art/security_cqc/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	if(check_streak(A,D))
+		return 1
+	log_combat(A, D, "grabbed (Security CQC)")
+	..()
```

Comment:
Something that would be *somewhat* more acceptable than forceful_disarm would be the ability to 'restrain' / instant aggressive grab anyone who is lying down. Although even then I'm still a bit eh on security_cqc in general even if weak.

Reply:
A better idea would be making this restrain stun both you and the criminal for some time, meaning while alone it is useless but if you have another sec officer with you, you can use it strategically.